validation/src/main/resources/config.json

```diff
@@ -0,0 +1,176 @@
+{
+  "$id": "/schema/config",
+  "type": "object",
+  "configItems": [
+    {
+      "key": "file_path",
+      "$ref": "classpath:/metadata-schema/baseSchema.schema.json#/properties/file_path",
+      "downloadFilesOutputs": [
+        {
+          "domain": "ClientTemplate",
```

Comment:
Is the ¬domain` value `ClientTemplate` a placeholder, or does it signify a specific use case, eg metadata download template?

Reply:
Changed it MetadataDownloadTemplate so more obvious what it applies to. Decided not to go with accumulative downloads base + closure as it introduced unnecessary complexity